src/dialog/marisa.c

```diff
@@ -326,10 +328,12 @@ DIALOG_TASK(marisa, Stage4PostBoss) {
 	MSG(marisa, "… but since when do I care about scientific facts?");
 
 	MSG(kurumi, "W-wait, what was I doing again?");
-	MSG(kurumi, "(And didn’t I see this nerdy looking girl before…?)");
+	MSG(kurumi, "(Y-yeah, that’s it, play it off as amnesia…");
+	MSG(kurumi, "Uuu, I don’t remember anything at all!!");
+	MSG(kurumi, "(But, hmm… haven’t I seen this nerdy-looking girl before…?)");
```

Comment:
Mostly she's trying to get out of trouble. In the previous version, she still had "amnesia" but I noticed I had it portrayed inconsistently between the different routes - where sometimes it was real, and sometimes it was being faked.

It's meant to show how silly and immature she is, since in another route she talks about being a big anime/manga fan, so it's a trope she's familiar with, and one she hopes will get her out of trouble.

Reply:
If it's for consistency's sake, then the change makes sense. To be honest I just find Kurumi to be awkwardly written. Maybe I don't quite understand her character. But that's out of scope for this PR, so LGTM for now.